.github/workflows/compilation.yml

```
@@ -0,0 +1,90 @@
+name: CI-compile
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: MechaPwn compilation
+    runs-on: ubuntu-latest
+    container: ps2dev/ps2dev:latest
+    # instead of "ps2dev/ps2dev:latest" you can use different tags, for example for old projects you can use "ps2dev/ps2dev:v1.0"
+    steps:
+    - name: Install dependencies
+      run: |
+        apk add build-base git zip python2 python2-dev 
+        apk add tiff-dev jpeg-dev openjpeg-dev zlib-dev freetype-dev lcms2-dev libwebp-dev tcl-dev tk-dev harfbuzz-dev fribidi-dev libimagequant-dev libxcb-dev libpng-dev
+        python -m ensurepip
+        pip install --upgrade pip setuptools wheel
+        pip install Pillow
+
+    - uses: actions/checkout@v2
+    - run: |
+        git fetch --prune --unshallow
+
+    - name: Compile project
+      run: |
+        make
+    # commands for compiling your project
+
+    - name: Get short SHA and repository name
+      id: slug
+      run: |
+        echo "::set-output name=sha8::$(echo ${GITHUB_SHA} | cut -c1-8)"
+        echo "::set-output name=REPOSITORY_NAME::$(echo "$GITHUB_REPOSITORY" | awk -F / '{print $2}' | sed -e "s/:refs//")"
+
+    - name: Upload artifacts
+      if: ${{ success() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ steps.slug.outputs.REPOSITORY_NAME }}-${{ steps.slug.outputs.sha8 }}
+        path: |
+          MechaPwn_pck.elf
+
+    # 'path' will create artifact with all *.elf in working directory
+    # you can change this
+  release:
+    needs: [build]
```

One item doesn't need brackets.
```suggestion
    needs: build
```